lang/ilias_de.lang

```diff
@@ -14881,3 +14881,8 @@ dateplaner#:#back_to_crs#:#Kursinhalt
 dateplaner#:#back_to_grp#:#Gruppeninhalt
 dateplaner#:#back_to_pd#:#Zum Persönlichen Schreibtisch
 crs#:#crs_timings_short_start_end#:#Startdatum
+lso#:#entries_updated#:#Einträge gespeichert
+lso#:#curriculum#:#Curriculum
+lso#:#unparticipate#:#Abmelden
+lso#:#participate#:#Anmelden
+lso#:#lso_player_unsubscribe#:#Abmelden
```

Comment:
Why do we have "Abmelden" twice here? I think we should just use the same lang-var in both places.